stacks/style.css

```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Stacks
+Theme URI: 
+Author: Automattic
```

Comment:
Maybe this is where we add the contributor's names? but it might be very long for some themes and this is what it shows here:

<img width="1465" alt="Screenshot 2023-03-09 at 16 37 00" src="https://user-images.githubusercontent.com/3593343/224074546-20e354ba-edda-41e6-bdcc-3fdaeed83b2d.png">